### PR TITLE
[ar] Translate index to arabic

### PR DESCRIPTION
**Pull request by mboukhalfa**
This PR adds the minimal translation related to the file _index.md and splitting the old https://github.com/kubernetes/website/pull/44902 to smaller

Part of https://github.com/kubernetes/website/issues/37546 and https://github.com/kubernetes/website/issues/44682

### CODE REVIEW

**Review by seifrajhi — COMMENTED** [review]
LGTM for Arabic

**Review by seifrajhi — APPROVED** [review]

**Comment by seifrajhi** [comment]
LGTM for Arabic

**Review by seokho-son — COMMENTED** [review]
PTAL: https://raw.githubusercontent.com/kubernetes/website/dev-1.29-ar.1/content/en/docs/_index.md

The header is different with the original English doc.
```
---
linktitle: Kubernetes Documentation
title: Documentation
sitemap:
  priority: 1.0
---
```

File: content/ar/docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: توثيق
```
Comment (adowair) [inline content/ar/docs/_index.md:2]: أرجح "الوثائق" أو "التوثيق"
```suggestion
title: الوثائق
```

Comment (mboukhalfa) [reply content/ar/docs/_index.md]: done

**Comment by adowair** [comment]
/lgtm

Thanks @mboukhalfa

**Pull request merged**